frontend_v2/src/app/components/utility/modal/modal.component.ts

```diff
@@ -75,6 +75,11 @@ export class ModalComponent implements OnInit {
    */
   deny = 'Cancel';
 
+  /**
+   * password feild visibility
+   */
+  passwordVisibility:String = 'password';
```

Comment:
@Alabhya268 instead of storing text `password` or `text` can we switch to a boolean `showPassword`

Reply:
ok